src/main.js

```diff
@@ -11,19 +11,24 @@ var coverImage = document.querySelector(".cover-image");
 var coverTitle = document.querySelector(".cover-title");
 var coverTagLine1 = document.querySelector(".tagline-1");
 var coverTagLine2 = document.querySelector(".tagline-2");
+
```

Comment:
I like that we have the spaces between our variable categories while we are working on the project. It reduces confusion, especially when we are getting tired! We'll need to remember to remove the blank lines before we submit the project.